test/jb_splitsStore/set.test.js

```diff
@@ -0,0 +1,293 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+import { abi as jbOperatorStoreAbi } from '../../artifacts/contracts/JBOperatorStore.sol/JBOperatorStore.json';
+import { abi as jbProjectsAbi } from '../../artifacts/contracts/JBProjects.sol/JBProjects.json';
+import { abi as jbDirectoryAbi } from '../../artifacts/contracts/JBDirectory.sol/JBDirectory.json';
```

Comment:
Let's follow the style of the other tests for now. We can change this pattern globally separately.